lib/r10k/source/svn.rb

```diff
@@ -106,6 +112,13 @@ def names_and_paths
 
     branches << [R10K::Environment::Name.new('production', opts), "#{@remote}/trunk"]
     @svn_remote.branches.each do |branch|
+      if !@branch_filter.nil? && !@branch_filter.empty?
```

Comment:
I think the same change as was made to git could be applied here. (Both the conditional change and the ignore_branch_prefix change).

Reply:
D'oh! It's been so long since I paid attention to svn in r10k.